tests/library/properties/property_test.go

```
@@ -0,0 +1,433 @@
+	a := assert.New(t)
+
+	for i := range schema {
+		schemaField := schema[i]
+		propertyField := properties[i]
+
+		a.Equal(schemaField.Fid, propertyField.Fid)
+		a.Equal(schemaField.Name, propertyField.Name)
+		a.Equal(schemaField.Type, propertyField.Type)
+		a.Equal(schemaField.Sort, propertyField.Sort)
+	}
+	return
```

golangci-lint (1.62.2)

S1023: redundant `return` statement

(gosimple)